recipes/streamlit-aggrid/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "streamlit-aggrid" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d96edb102df98450f3b6bcf5a52b64e6ceef9678e82e2dba9cfef074e7f134ab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.7,<4.0
+  run:
+    - pandas >=1.2
```

Comment:
Upstream lists:
```ini
    install_requires=[
        "streamlit >= 0.75",
        "simplejson >= 3.0",
        "pandas",
        "numpy"
```
Is there any reason this recipe doesn't match?

Reply:
im using the pyproject requirements
https://github.com/PablocFonseca/streamlit-aggrid/blob/main/pyproject.toml#L14

Reply:
Alright then. 🤷‍♂️